velox/functions/prestosql/fuzzer/ApproxPercentileResultVerifier.h

```diff
@@ -106,6 +134,13 @@ class ApproxPercentileResultVerifier : public ResultVerifier {
         return false;
       }
     }
+    if (verifyWindow_ && numNonNull != allRanges_->size()) {
```

Comment:
do you need the condition on verifyWindow_? I think this would have caught the bug I fixed in my previous change to this class where we were losing the first percentile

Reply:
Yes, we do need verifyWindow_ here because for the verification for aggregation fuzzer, the number of non-null rows in result doesn't necessarily match the number of non-null rows in allRanges_, so we cannot compare them.